src/test/java/com/aparapi/codegen/test/ReturnBooleanNewArray.java

```diff
@@ -26,4 +26,4 @@ public void run() {
         returnBooleanNewArray();
     }
 }
-/**{Throws{ClassParseException}Throws}**/
+/**{Throws{CodeGenException}Throws}**/
```

Comment:
Isnt this PR adding array creation support? Any reason boolean arrays arent supported?

Reply:
Nevermind this is unrelated to your commit (an issue with aparapi we have to resolve). I see below you do support creating new booleans they just arent returned on the same line.

Reply:
I was not able to find OpenCL syntax for java `return new boolean[1024];`

Reply:
@shabanovd i added this as an issue with a $25 bounty, so hopefully someone will find a solution. I when i have time i may be able to look into it.